packages/blade/src/components/Dropdown/Dropdown.tsx

```diff
@@ -101,9 +100,6 @@ const _Dropdown = ({
 
   useDidUpdate(() => {
     onOpenChange?.(isOpen);
-    if (!isOpen) {
```

Comment:
Why was this removed?

Reply:
`onDismiss` prop was deprecated as we later added `onOpenChange`. The deprecated prop is removed.